### PR TITLE
KFSPTS-10033 implement cloudcheckr POJOs

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    @XmlJavaTypeAdapter(KualiDecimalXmlAdapter.class)
+    private KualiDecimal minimum;
+    
+    @XmlElement(name = "Average", namespace = StringUtils.EMPTY)
+    @XmlJavaTypeAdapter(KualiDecimalXmlAdapter.class)
+    private KualiDecimal average;
+    
+    @XmlElementWrapper(name = "CostsByGroup", namespace = StringUtils.EMPTY)
+    @XmlElement(name = "GroupingLevel", namespace = StringUtils.EMPTY, required = true)
+    private List<GroupLevel> groupLevels;
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java]: I am wondering if 'costsByTime' may be a better name for this list?

File: src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    
+    @XmlElementWrapper(name = "CostsByTime", namespace = StringUtils.EMPTY)
+    @XmlElement(name = "GroupingByTime", namespace = StringUtils.EMPTY, required = true)
+    private List<GroupingByTime> groupingByTimes;
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java]: I am wondering if 'costsByTime' may be a better name for this list?

File: src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        groupingByTimes.stream().forEach(costtime -> sb.append("costByTime: (").append(costtime.toString()).append("), "));
+        return sb.toString();
+    }
+    
+    @Override
+    public boolean equals(Object o) {
+        boolean equals = false;
+        if (ObjectUtils.isNotNull(o) && o instanceof CloudCheckrWrapper) {
+            CloudCheckrWrapper otherWrapper = (CloudCheckrWrapper) o;
+            equals = total.equals(otherWrapper.total) && maximum.equals(otherWrapper.getMaximum())
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java]: Should this code also check for null on total, maximum, minimum, average or maybe use Objects.equals for comparison on these fields?

File: src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/GroupLevel.java [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+    public void setNextLevel(List<GroupLevel> nextLevel) {
+        this.nextLevel = nextLevel;
+    }
+    
+    @Override
+    public boolean equals(Object o) {
+        boolean equals = false;
+        if (ObjectUtils.isNotNull(o) && o instanceof GroupLevel) {
+            GroupLevel otherGl = (GroupLevel) o;
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/GroupLevel.java]: otherGl should be changed to otherGroupLevel

File: src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java (hunk 1 above)
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java]: How about costsByGroup? .... sure I can change this

File: src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java (hunk 2 above)
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java]: sure

File: src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java (hunk 3 above)
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java]: Yes, thank you, I missed these fields

File: src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/GroupLevel.java (hunk 4 above)
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/GroupLevel.java]: sure

File: src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java (hunk 1 above)
Comment (db483) [reply src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/CloudCheckrWrapper.java]: that is what I meant but must have copy pasted the one below this. thank you.

**Comment by jhulslander** [comment]
This is ready for review agian

**Pull request merged**